src/components/one-time-donation/components/FormikStepper.tsx

```diff
@@ -50,10 +51,18 @@ export function FormikStepper<T>({ children, ...props }: GenericFormProps<T>) {
   const [step, setStep] = useState(0)
   const currentChild = childrenArray[step]
   const classes = useStyles()
-
+  const { data: currentPerson } = useCurrentPerson()
   function isLastStep() {
     return step === childrenArray.length - 2
   }
+
+  function isFirstStep() {
+    return step === 0
+  }
+
+  function isLogged() {
+    return currentPerson && currentPerson.status !== 'unauthenticated'
```

Comment:
This is not really safe as the `status` can be `undefined` or empty string and they still would be considered `logged`